Snakefile

```diff
@@ -185,6 +185,22 @@ if config['enable'].get('retrieve_natura_raster', True):
         run: move(input[0], output[0])
 
 
+if config['enable'].get('build_ship_raster', False):
+    rule build_ship_raster:
+        input:
+            ship_density="data/bundle/shipdensity/shipdensity_global.zip",
+            cutouts=expand("cutouts/{cutouts}.nc", **config['atlite'])
+        output: "resources/europe_shipdensity_raster.nc"
+        log: "logs/build_ship_raster.log"
+        script: "scripts/build_ship_raster.py"
+
+
+if config['enable'].get('retrieve_ship_raster', True):
+    rule retrieve_ship_raster:
+        input: HTTP.remote("path-to-file", keep_local=True, static=True)
```

Comment:
Can you upload to `sandbox.zenodo.org` (datasets published there will be deleted each day)? Then CI will even pass.